scripts/travis_script

```diff
@@ -20,6 +20,7 @@ case "$TASK" in
             export JAVA_HOMES="$(jdk_switcher home openjdk6)/jre:$(jdk_switcher home openjdk7)/jre:$(jdk_switcher home oraclejdk8)/jre:$(jdk_switcher home oraclejdk9)"
             ./gradlew clean assemble --stacktrace
             ./gradlew check :opencensus-all:jacocoTestReport
+            ./gradlew verGJF
```

Comment:
Could you please add a note about the new check and gradle commands to CONTRIBUTING.md?

Reply:
Filed an issue https://github.com/census-instrumentation/opencensus-java/issues/773